README.md

```diff
@@ -1,24 +1,44 @@
 Deriv App
 ============
-This repository contains the various platforms of Deriv.
+This repository contains the various platforms of the Deriv application.
 
+### In this document:
 - [Installation](#installation)
+- Packages in this repo:
+  - [Bot](packages/bot/README.md)
+  - [Components](packages/components/README.md)
+  - [Core](packages/core/README.md)
+  - [P2P](packages/p2p/README.md)
+  - [Shared](packages/shared/README.md)
+  - [Trader](packages/trader/README.md)
+  - [Translations](packages/translations/README.md)
 - [Working With This Repo](#working-with-this-repo)
+    - [Package names](#package-names)
 - [Usage](#usage)
+  - [Starting a Development Server](#starting-a-dev-server)
   - [How to Clean Packages](#how-to-clean-packages)
-  - [Examples](#examples)
+  - [Examples of Script Usage](#examples-of-script-usage)
 - [PR Guidelines](#pr-guidelines)
 - [FAQ](#faq)
 
+### Other documents:
+- [General](docs/README.md) - Contains general philosophy and overview of this package
+- [Stylesheet guidelines](docs/Stylesheet/README.md) - Contains rules for CSS/SASS code style
+- [JavaScript guidelines](docs/JavaScript/README.md) - Contains rules for JS/JSX code style
+- [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
+
 [comment]: <> (TODO: Refactor Clean Project to be under usage)
 
 ## Installation
-In order to work on your own version of the Deriv Javascript and CSS, please **fork this project**.
+In order to work on your own version of the Deriv application, please **fork this project**.
 
 You will need to perform the following on your development machine:
 
-1. Node.js (10.14.2 or higher is recommended) and NPM (see <https://nodejs.org/en/download/package-manager/>)
-2. Run `npm run bootstrap` from the root folder
+1. Node.js (v11.15.0 is recommended, v12 has compat issues currently) and NPM (see <https://nodejs.org/en/download/package-manager/>)
```

Comment:
Random Question#1: Did the problem with `lerna bootstrap` throwing errors in node v11 and higher resolved?

Reply:
@reza-fs We have issues with lerna v12. Today I spoke with @mahdiaryayi-fs and he showed the issue to me for v11, and it **seems** to be the issue I have addressed in the FAQ section, question #4. So v11 should be safe (v11.15.0 is definitely safe).